lms/djangoapps/courseware/views/views.py

```diff
@@ -1138,6 +1138,21 @@ def _credit_course_requirements(course_key, student):
     }
 
 
+def _course_home_redirect_enabled():
+    """
+    Return True value if user needs to be redirected to course home based on value of
+    `ENABLE_MKTG_SITE` and `ENABLE_COURSE_HOME_REDIRECT feature` flags
+
+    Returns: boolean True or False
+    """
+    if configuration_helpers.get_value(
+            'ENABLE_MKTG_SITE', settings.FEATURES.get('ENABLE_MKTG_SITE', False)
+    ) and configuration_helpers.get_value(
+        'ENABLE_COURSE_HOME_REDIRECT', settings.FEATURES.get('ENABLE_COURSE_HOME_REDIRECT', True)
```

Comment:
is the default for `ENABLE_COURSE_HOME_REDIRECT = True` expected or should it be `False` like `ENABLE_MKTG_SITE `?

Reply:
nit: I think it would be better to explicitly return `False` at the end instead of the default `None` to avoid any future misuses of the function.

Reply:
default for `ENABLE_COURSE_HOME_REDIRECT` is set to `True` to avoid breaking existing behavior after this change.
returning `False` in `_course_home_redirect_enabled` cause this [quality violation](https://build.testeng.edx.org/job/edx-platform-quality-pipeline-pr/411/Diff_20Quality_20Report/).